How-to-contribute.md

```diff
@@ -1,22 +1,20 @@
-<h1> HOW TO CONTRIBUTE </h1>
+<h1> How To Contribute & Earn XUC </h1>
 
-Thank you for considering to help in building Exchange Union! We welcome any contributions no matter how big or small - from typos to bug fixes, you will be awarded with XUC (Exchange Union's token) after carefully analyzing every pull request. Github is our git version control service of choice, used to track issues, suggestions, feature request, documentations and is home to the source code which powers Exchange Union. 
+Yay! Kudos for being a part of Exchange Union & help building XUD, our node software! Any contributions are welcome no matter how big or small. From typos and bug fixes to entire modules - you will be awarded with XUC, our token.
 
-If you'd like to contribute to the Exchange Union, please fork our repository, push your commits or fixes and send a pull
-request for us to review and merge into the development branch. If, however, you wish to submit substaintial changes or features -
-We encourage you to first have a chat with us on our [Slack Channel](https://exchangeunion.slack.com/).
+Here is how it works:
+1. Grab a task from [here.](https://github.com/ExchangeUnion/xud/issues)
+2. Each task carries a range of XUC to be earned, like e.g. 50-100 XUC. The higher the reward, the more complex - grab a task which best fits your time at hand & skill set.
+3. Let everyone know via the comment field (!) and fork the repo to start workin on your task. 
+4. [Get in touch](https://gitter.im/exchangeunion/Lobby) for help on the way, especially for testing. Our automation is still in the works :/
+5. When you feel ready, submit a pull request  against the `development` branch with your <b>ether address (!)</b> in the comment section
```

Comment:
Don't see a reason why not to open the PR against `master`

Reply:
Yes, it should be against master.

Reply:
ok, changed